.github/workflows/template.module.yml

```diff
@@ -25,27 +25,27 @@ env:
   TOKEN_NAMEPREFIX: '${{ secrets.TOKEN_NAMEPREFIX }}'
 
 jobs:
-  #########################
-  #   Static validation   #
-  #########################
-  job_module_static_validation:
-    name: 'Static validation'
-    runs-on: ubuntu-20.04
-    if: (fromJson(inputs.workflowInput)).staticValidation == 'true'
-    steps:
-      - name: 'Checkout'
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Set environment variables
-        uses: ./.github/actions/templates/setEnvironmentVariables
-        with:
-          variablesPath: ${{ env.variablesPath }}
-      - name: 'Run tests'
-        uses: ./.github/actions/templates/validateModulePester
-        with:
-          modulePath: '${{ inputs.modulePath }}'
-          moduleTestFilePath: '${{ env.moduleTestFilePath }}'
+  # #########################
```

Comment:
I guess this should be undone. I'd would have anyways suggested to simply disable the deployment test with the checkbox ;)

Reply:
Bad habit ;) will undo

Reply:
Undone, and now a question arises: Should the PSRule job be skipped when the staticValidation input parameter is unchecked? 🤔

Reply:
Yes. I guess they would be in one stage if technically possible - but it isn't for ir GH

Reply:
done